pages/blog/blog-post.js

```diff
@@ -137,6 +137,47 @@ const SocialShareLink = SocialLink.withComponent('div').extend`
   margin: 0;
 `
 
+const AuthorSection = ({
```

Comment:
This could be a component in a different file. But I see there a few more components in this file so it is fine.

Reply:
Yes, this whole file could be split up really. But for a different PR...